privacy/views/privacy_activity_view.xml

```diff
@@ -33,28 +33,34 @@
                     <group name="basic">
                         <group name="owners">
                             <field name="controller_id"/>
-                            <field
-                                name="processor_ids"
-                                widget="many2many_tags"
-                            />
-                        </group>
-                        <group name="subjects">
-                            <field name="subject_find"/>
-                            <field
-                                name="subject_domain"
-                                widget="char_domain"
-                                options='{"model": "res.partner"}'
-                                attrs='{"required": [("subject_find", "=", True)],
-                                        "invisible": [("subject_find", "=", False)]}'
-                            />
                         </group>
                     </group>
                     <notebook string="Details" name="advanced">
-                        <page string="Description">
+                        <page string="Description" name="description">
                             <group>
                                 <field name="description" nolabel="1"/>
                             </group>
                         </page>
+                        <page string="Subjects" name="subjects">
+                            <group name="subjects">
+                              <field name="subject_find"/>
+                              <field
+                                  name="subject_domain"
+                                  widget="domain"
+                                  options='{"model": "res.partner"}'
+                                  attrs='{"required": [("subject_find", "=", True)],
+                                          "invisible": [("subject_find", "=", False)]}'
+                              />
+                            </group>
+                        </page>
+                        <page string="Processors" name="processors">
```

Comment:
Doing a page only for this field is too much IMO.

Reply:
This is the base module but some other will come and will have more option for processors, like tracking if they are GPDR compliant, extra EU, etc ..
That's why I proposed this.

Reply:
I don't like too much any way. What original promoters of this module - @Yajo @mreficent - think?

Reply:
For exemple I need to add purpose, storage medium used, if processors are Extra EU, need to track legal documents etc ..

I understand your comment, think we can have a good solution.